pkg/providers/aws/provider_postgres.go

```diff
@@ -203,6 +202,77 @@ func (p *AWSPostgresProvider) createRDSInstance(ctx context.Context, cr *v1alpha
 		}
 		return nil, "modify instance in progress", nil
 	}
+	// Add Tags to Aws Postgres resources
+	logrus.Infof("Adding Tags to RDS instance %s", *foundInstance.DBInstanceIdentifier)
+	// get the environment from the CR
+	defaultOrganizationTag := cr.Spec.Env.Value
+
+	// new session but don't need credentials
+	// Get the region from AvailabilityZone
+	rdsRegion := *foundInstance.AvailabilityZone
+	regionLen := len(rdsRegion)
```

Comment:
Does this need to be a variable?

Reply:
probably not but I feel adding a variable makes it easier to read.